alerts/openshift-virtualization-operator/HCOMisconfiguredDescheduler.md

```diff
@@ -0,0 +1,70 @@
+# HCOMisconfiguredDescheduler
+
+## Meaning
+
+A Descheduler is a OpenShift Container Platform application that causes the
+control plane to
+re-arrange the workloads in a better way.
+
+The descheduler uses the OpenShift Container Platform eviction API to evict
+pods, and receives
+feedback from `kube-api` whether the eviction request was granted or not.
+On the other side, in order to keep VM live and trigger live-migration,
+OpenShift Virtualization handles eviction requests in a custom way and
+unfortunately
+a live migration takes time.
```

Comment:
-> In contrast, to keep a VM live and trigger a live migration, OpenShift Virtualization handles eviction requests in a custom manner, and a live migration takes time to perform.